src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt

```diff
@@ -1,6 +1,6 @@
 version := "0.1"
 
-scalaVersion := "3.5.0"
+scalaVersion := "3.3.4"
```

Comment:
I can definitely add a separate test for this. Since this is just a flag that is passed to the compiler, I feel using the minimum version that supports the flag makes sense.

Reply:
That used to be 3.4.2 and we had a bug where 3.5.0 was not working properly. So that was updated to check that the fix works.

Reply:
Got it, I'll revert this and add a separate test for 3.3.4